valuesets/source-update/update-rat.sh

```diff
@@ -0,0 +1,10 @@
+curl https://covid-19-diagnostics.jrc.ec.europa.eu/devices/hsc-common-recognition-rat |
+jq '.extracted_on as $version | .deviceList | reduce .[] as $i
+({};
+.[$i.id_device] =
+{ "display": ($i.manufacturer.name + ", " + $i.commercial_name),
+    "lang":"en",
+    "active": true,
+    "system": "https://covid-19-diagnostics.jrc.ec.europa.eu/devices",
+    "version": $i.last_updated
+}) | {"valueSetId": "covid-19-lab-test-manufacturer-and-name", "valueSetDate":$version[0:10], "valueSetValues": .}' > ~/dev/GitHub/ehn-dgc-schema/valuesets/test-manf.json
```

Comment:
Remove output redirect from script ( `> ~/ ...`) and allow caller to pipe / redirect at shell level, if required.

Reply:
Would it be ok, to mark it as a comment?

#> ~/dev/GitHub/ehn-dgc-schema/valuesets/test-manf.json

Reply:
I would remove it and create a makefile target that uses this script ot update the in-repository files

Reply:
Guys, could you do it? I am afraid that this goes beyond my programming experience.

Reply:
yes, will modify in-situ, don't need a makefile yet - T-Systems will basically take this script and use it for updates, up to them how they embed it